crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -128,7 +128,7 @@ pub fn transaction_request_to_typed(
             }))
         }
         // EIP4844
-        (Some(3), None, _, _, _, Some(_), Some(_), Some(sidecar), to) => {
+        (Some(3), None, _, _, _, _, Some(_), Some(sidecar), to) => {
```

Comment:
This change is still necessary to parse the transaction correctly, this fixes the `called `Result::unwrap()` on an `Err` value: ErrorResp(ErrorPayload { code: -32602, message: "Failed to decode transaction", data: None })` error